packages/mui-utils/src/integerPropType.d.ts

```diff
@@ -0,0 +1,4 @@
+import { Requireable } from 'prop-types';
+
+declare const integerPropType: Requireable<number>;
```

Comment:
TS was complaining that `integerPropType` had no typing file when I moved to `composite` build